policies/policy_service_test.go

```diff
@@ -1251,6 +1251,79 @@ func TestInactivateDatasetByID(t *testing.T) {
 	}
 }
 
+func TestDeleteAGroupFromDataset(t *testing.T) {
```

Comment:
we could try to see if we have a stubbing or spy on the logs, to fetch those execution logs